src/crimson/CMakeLists.txt

```diff
@@ -85,6 +85,7 @@ add_library(crimson-common STATIC
   ${PROJECT_SOURCE_DIR}/src/common/PluginRegistry.cc
   ${PROJECT_SOURCE_DIR}/src/common/RefCountedObj.cc
   ${PROJECT_SOURCE_DIR}/src/common/util.cc
+  ${PROJECT_SOURCE_DIR}/src/compressor/Compressor.cc
```

Comment:
this breaks the build. see https://shaman.ceph.com/builds/ceph/wip-kefu-testing-2020-09-04-1852/da394fa67e09acd45ad77a597bd9a9191845381d/crimson/227393/

crimson-alienstore already includes `compressor_objs`

see https://github.com/ceph/ceph/blob/b5e47e7bfaa4717cee5b808546260b9484c585ae/src/crimson/os/alienstore/CMakeLists.txt#L44-L46

Reply:
I've fixed it, can you please have a look?

Reply:
@tchaikov

Reply:
@giladmaya i cannot find the changelog of the latest revision. will just drop my ack anyway.